packages/amplify_auth_cognito/android/src/main/kotlin/com/amazonaws/amplify/amplify_auth_cognito/AuthCognito.kt

```diff
@@ -347,6 +373,46 @@ public class AuthCognito : FlutterPlugin, MethodCallHandler {
     }
   }
 
+  private fun onSignInWithWebUI(@NonNull flutterResult: Result, @NonNull request: HashMap<String, *>) {
+    if (FlutterSignInWithWebUIRequest.validate(request)) {
+      var req = FlutterSignInWithWebUIRequest(request)
+      var resultSubmitted: Boolean = false;
```

Comment:
Is this needed because somehow the success callback of `Amplify.Auth.signInWithWebUI` is being called twice?

Reply:
Correct (althought it's possible that this was fixed, so I'll confirm that it's still needed).